src/widgets/attachments/attachments-overview.css

```diff
@@ -7,3 +7,12 @@
   padding: 6px 12px;
   cursor: pointer;
 }
+
+.cameraSection {
+  padding: 10px;
+}
+
+.react-html5-camera-photo > img,
```

Comment:
Are these working? I thought the class names are prefixed and do not apply.

Reply:
Sorry those aren't working. That was why I created a global style and put them there. I must have forgotten to delete here.